website/language.py

```diff
@@ -72,6 +72,21 @@
 
 INVALID_EMAIL_CONFIRM_TOKEN = 'This confirmation link is invalid. Please <a href="/login/">log in</a> to continue.'
 
+CANNOT_MERGE_ACCOUNTS_SHORT = 'Cannot Merge Accounts'
+
+CANNOT_MERGE_ACCOUNTS_LONG = 'Accounts cannot be merged due to a conflict.'
```

Comment:
Is this what the user sees if there are addons that can't be merged? At the very least, the user should be given what action needs to be taken before merging.